Runtime/McSubscriber.cs

```diff
@@ -200,11 +208,47 @@ private void EstablishSignalingConnection(string ws_url, string token)
                     case ISignaling.Event.VIEWER_COUNT:
                         OnViewerCount?.Invoke(this, payload.viewercount);
                         break;
+                    case ISignaling.Event.LAYERS:
+                        try
+                        {
+                            simulCastInfo = DataContainer.ParseSimulcastLayers(payload.medias);
+                            OnSimulcastlayerInfo?.Invoke(this, simulCastInfo);
+                            simulcastEvent?.Invoke();
```

Comment:
simulcastInfo & the subscriber need to be passed to this event callback, how else are users of the editor API going to know the layers? We also need to add a tooltip to explain the `simulcastEvent` callback better.

Reply:
Right, if the user wants to get the simulcast layer info, it needs to be registered to the delegate: OnSimulcastlayerInfo
which will send the subscriber and simulcastInfo.
We cannot send the parameters to UI event as its completely depends on which method user wants to trigger, by attaching reference to the gameobject and choosing the public method from inspector.